base/markdown/render/rich.jl

```diff
@@ -3,7 +3,7 @@ function tohtml(io::IO, m::MIME"text/html", x)
 end
 
 function tohtml(io::IO, m::MIME"text/plain", x)
-    writemime(io, m, x)
+    printesc(io, sprint(writemime, m, x))
```

Comment:
@one-more-minute  is there a  way to do this line without sprint-ing (i.e. in one pass)?

Any other comments ?